src/index.html

```diff
@@ -7,12 +7,71 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__product-image"
+        src="./images/imac.jpeg"
+        alt="product-image"
+      />
+
+      <div class="card__product-details">
+        <p class="card__product-details-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__product-details-description">Product code: 195434</p>
+      </div>
+
+      <div class="card__rating">
+        <div class="stars stars--4">
```

Comment:
The class 'stars--4' implies a modifier that should change the appearance of the stars based on the rating. Ensure the corresponding SCSS uses this modifier to reflect the correct rating visually.